src/Output/Table.php

```diff
@@ -23,11 +23,11 @@
 use function implode;
 use function is_array;
 use function max;
+use function mb_strwidth;
```

Comment:
caution: mb_ funcs may not present in all php installations (comes from mbstring extension)

so we need a fallback to just plain strwidth/substr etc in that case

Reply:
done